### DISCUSSION TITLE
Creating connection using DynamoDB and Bigquery

### DISCUSSION THREAD

**Discussion by payalevonence** [airbytehq/airbyte#34667] [Connector Questions]
Hello, I am facing Discovering schema failed common. error while connecting DynamoDB to BigQuery using Airbyte
I have successfully set the source as well as the destination, but while creating the connection it is getting an error.
Please help me with this issue.

**Comment by sajarin** [comment]
hey, what version of airbyte and connectors are you using?

**Reply by sasharevzin** [reply]
I think it is related to this https://github.com/airbytehq/airbyte/issues/25952
It happens with Airbyte v0.50.42 with the latest DynamoDB and Bigquery connectors

**Comment by ayanguas** [comment]
Same error here. Were you able to fix it?

[a comment by an automated account is not shown]
